sidebarLearn.json

```
@@ -1,369 +1,391 @@
 [
+	{
+			"title": "What is meilisearch",
+			"slug": "what_is_meilisearch",
+			"routes": [
+					{
+						"source": "learn/what_is_meilisearch/overview.mdx",
+						"label": "Overview",
+						"slug": "overview"
+					},
+					{
+						"source": "learn/what_is_meilisearch/features.mdx",
+						"label": "Features",
+						"slug": "features"
+					},
+					{
+						"source": "learn/what_is_meilisearch/philosophy.mdx",
+						"label": "Philosophy",
+						"slug": "philosophy"
+					},
+					{
+							"source": "learn/what_is_meilisearch/sdks.mdx",
+							"label": "Official SDKs/libraries",
+							"slug": "sdks"
+					},
+					{
+							"source": "learn/what_is_meilisearch/comparison_to_alternatives.mdx",
+							"label": "Comparison to alternatives",
+							"slug": "comparison_to_alternatives"
+					},
+					{
+							"source": "learn/what_is_meilisearch/telemetry.mdx",
+							"label": "Telemetry",
+							"slug": "telemetry"
+					},
+					{
+							"source": "learn/what_is_meilisearch/search_preview.mdx",
+							"label": "Search preview",
+							"slug": "search_preview"
+					},
+					{
+							"source": "learn/what_is_meilisearch/language.mdx",
+							"label": "Language",
+							"slug": "language"
+					},
+					{
+							"source": "learn/what_is_meilisearch/contact.mdx",
+							"label": "Contact",
+							"slug": "contact"
+					}
+			]
+	},
+	{
+			"title": "Getting started",
+			"slug": "learn/getting_started",
+			"routes": [
+					{
+							"source": "learn/getting_started/installation.mdx",
+							"label": "Installation",
+							"slug": "installation"
+					},
+					{
+							"source": "learn/getting_started/supported_os.mdx",
+							"label": "Supported operating systems",
+							"slug": "supported_os"
+					},
+					{
+							"source": "learn/getting_started/quick_start.mdx",
+							"label": "Quick start",
+							"slug": "quick_start"
+					},
+					{
+							"title": "Meilisearch 101",
+							"slug": "learn/getting_started",
+							"routes": [
+									{
+											"source": "learn/getting_started/filtering_and_sorting.mdx",
+											"label": "Filtering and sorting",
+											"slug": "filtering_and_sorting"
+									},
+									{
+											"source": "learn/getting_started/customizing_relevancy.mdx",
+											"label": "Customizing relevancy",
+											"slug": "customizing_relevancy"
+									},
+									{
+											"source": "learn/getting_started/getting_ready_for_production.mdx",
+											"label": "Getting ready for production",
+											"slug": "getting_ready_for_production"
+									}	
+							]
+					}
+			]
+	},
   {
       "title": "Core concepts",
       "slug": "core",
```

```suggestion
      "slug": "core_concepts",
```

So that we don't have to fix links